package.json

```diff
@@ -40,7 +40,11 @@
   "commitlint": {
     "extends": [
       "@commitlint/config-conventional"
-    ]
+    ],
+    "rules": {
+      "header-max-length": [0],
+      "body-max-line-length": [0]
+    }
```

Comment:
I need to disable them for now due to the commit message of the 1st release. We will reable them just after the next release ;)